### DISCUSSION TITLE
Latest Update / Linux-Tools

### DISCUSSION THREAD

**Discussion by Nationalgeographic** [Slimbook-Team/slimbookbattery#71] [Q&A]
Hey guys, 
after updating Slimbookbattery from the AUR (I'm using Manjaro) I get this prompt:
"You have not installed linux-tools for your kernel version, which is recommended. If you want to install it, click the 'Install' button below, otherwise, you can close this window, and everything will remain the same."

Is the name of the corresponding package "linux-tools-meta"? I don't know which package is referred to exactly. Because the "install" button does not work in Manjaro.

**Comment by Flashwalker** [comment]
If you were an ubuntu user, you would need this package:
```sh
apt install linux-tools-$(uname --kernel-release)
```
